gravis3d/generator.py

```diff
@@ -12,15 +14,45 @@ def __init__(self, N = None):
         vx,vy,vz = normal(0,1e7,N), normal(0,1e7,N), normal(0,1e7,N)
         r = abs(normal(1e9,1e8,N))
         self.bodies = []
-        for i in range(N):
+        for i in range(0,N):
```

Comment:
Why this was required? No need to change

Reply:
ya sure